jiant/utils/utils.py

```diff
@@ -418,10 +418,7 @@ def format_output(obj, cuda_devices):
 
 
 def uses_cuda(cuda_devices):
-    use_cuda = 1
-    if isinstance(cuda_devices, list) or isinstance(cuda_devices, int) and cuda_devices >= 0:
-        return use_cuda
-    return 0
+    return isinstance(cuda_devices, list) or isinstance(cuda_devices, int) and cuda_devices >= 0
```

Comment:
me being paranoid: add parens to the latter two statements ?

but yaaaay for succinctness